composer.json

```diff
@@ -17,6 +17,7 @@
         "psr/http-server-middleware": "^1.0"
     },
     "require-dev": {
+        "cakephp/authentication": "2.x-dev",
```

Comment:
Should this also be in `suggest` for if someone wants to use the `Identity` class in this plugin?

Reply:
I think only someone already using authentication would be interested in this class.

The new class doesn't provide anything which would make someone chose this class first and then make them use authen. lib.